src/CommentBox/CommentBox.js

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import CommentList from './CommentList';
+import CommentForm from './CommentForm';
+import style from './style';
+
+class CommentBox extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { data: [] };
+    this.loadCommentsFromServer = this.loadCommentsFromServer.bind(this);
+    this.handleCommentSubmit = this.handleCommentSubmit.bind(this);
+    this.handleCommentDelete = this.handleCommentDelete.bind(this);
+    this.handleCommentUpdate = this.handleCommentUpdate.bind(this);
+    this.pollInterval = null;
+  }
+  loadCommentsFromServer() {
+    axios.get('http://localhost:3001/api/comments')
```

Comment:
Will be better to extract the url into constant variable. And after deploying the code to heroku I suppose that the code will not work.